272 auto expiry for events

Thrillberg opened the pull request.

Thrillberg commented:
@groe Here is a (broken) PR for the feature. I have some notes detailed in the 2nd commit about what's missing. Let me know if this helps!

groe commented:
Thank you, @Thrillberg! It could take a few days unfortunately but I will look into it and let you know!

groe commented on config/locales/views.de.yml:
```diff
@@ -25,6 +25,28 @@ de:
+      first_paragraph: |
+        We have unpublished your ongoing event. If you would like to visit your event, "%{ongoing_event_title}", please click
+      event_link: "here."
+      second_paragraph: "Thank you for publishing your event with us!"
```
I can add the german translations. Ok?

Thrillberg commented:
@groe I've (finally!) made changes to my PR that are hopefully an improvement. It's still not working perfectly (some links to the `show` view are broken and I couldn't really figure out why) but I've addressed your comments above.
